lib/confluence/metric_computer.es6.js

```diff
@@ -42,87 +42,100 @@ foam.CLASS({
     },
     {
       name: 'mlang',
-      documentation: `Object contains mlang expressions, it supports
-        logic operations such as AND and OR and math operations such
-        as IN, EQ.`,
+      documentation: `The mlang singleton expression.`,
       factory: function() {
         return this.ExpressionsSingleton.create();
       },
     },
   ],
   methods: [
     {
+      name: 'getOrderedListOfBrowserReleaseDates',
+      documentation: `Get an array of Dates for which there is a release
+          for any of major browsers.`,
+      returns: {
+        typeName: 'Date[]',
```

Comment:
This isn't quite true, right? The `groupKeys` that you return will be an array of Strings?